src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -248,6 +250,12 @@ public static System.Type GetImportedType(string typename)
 
             return null;
         }
+
+        /// <summary>
+        /// If true reflection methods will be used to get module types
+        /// </summary>
+        internal static bool IsReflectionContext { get; set; }
```

Comment:
can you amend this comment to include the use case - why would someone use reflection to get module types, and explain how it's used in the markdown docs CLI tool.